frontends/web/src/components/rates/rates.tsx

```diff
@@ -99,6 +99,9 @@ interface ProvidedProps {
     unstyled?: boolean;
     skipUnit?: boolean;
     noAction?: boolean;
+    typeClassName?: string;
+    sign?: string;
```

Comment:
how about `sign?: '+' | '-'`?

then at this point you can remove `typeClassName` here and move the style.send and style.receive from transaction.tsx to this component instead. would be a lot simpler to understand imo than the current code, where the logic is quite hard to understand at a glance.

Reply:
Seems really nice, but the style is also used outside the `FiatConversion` component, to render the coin amounts (e.g. https://github.com/digitalbitbox/bitbox-wallet-app/blob/master/frontends/web/src/components/transactions/transaction.tsx#L211). Maybe I can refactor `typeClassName` to `txTypeStyle` or smth similar, to make it more clear. wdyt?

Reply:
> the style is also used outside the FiatConversion component

Is this about the color? It would be nice if the `FiatConversion` component would behave like text and just inherit the parents color.

Reply:
oh too bad. yeah or maybe just a comment above the prop

Reply:
@thisconnect the point is that if there is on value to show, it should show `---` or `N/A` without style (otherwise it would be green/red). At least that's how I understood it.

Reply:
Ah I see, in case for `N/A` the FiatComponent could apply normal text color and overwrite the parents color.

Reply:
Now should be better: the style is inherited by the parent, unless the amount is not available.

Reply:
> how about `sign?: '+' | '-'`

also tried this, and it looks like the share decorator messes something up with the types. I propose to keep as string for now and then change it after rates.tsx is refactored.